qcodes/instrument_drivers/rigol/DG1062.py

```diff
@@ -160,8 +165,8 @@ def __init__(self, parent: 'DG1062', name: str, channel: int) ->None:
                 ),
                 vals.Enum("INF", "MIN", "MAX", "HighZ")
             ),
-            set_parser=lambda value: "INF" if value == "HighZ" else value,
             get_parser=lambda value: "HighZ"
+            set_parser=lambda value: "INF" if value == "HighZ" else value,
             if float(value) > DG1062Channel.max_impedance else float(value)
         )
```

Comment:
something terribly wrong happened here :) the automatic testing is alarming: https://ms-quantum.visualstudio.com/QubitWorkbench/_build/results?buildId=10600.

I think when moving `set_parser` inserted it in the middle of the lambda expression for `get_parser` which breaks everything :) I guess the following is what you wanted, right?:
```python
<...>
),  # <- comma from the line before
get_parser=lambda value: "HighZ" 
    if float(value) > DG1062Channel.max_impedance else float(value), # <- comma
set_parser=lambda value: "INF" if value == "HighZ" else value  # <- no comma here because of the brace on the next line
)
<...>
```

Also, you sure this final version of the code has been fully tested on a real device? :)

Reply:
Yes, this caused a problem. Thanks for picking it up